### DISCUSSION TITLE
How to paste yanked text in prompts

### DISCUSSION THREAD

**Discussion by KalinDimitrow** [helix-editor/helix#8959] [Troubleshooting] [answered]
I want to learn how to use helix and start using it as my development environment.
I literally can't find a way how to paste in prompts, can you add this information somewhere or if it's missing feature can you add it

**Comment by kirawi** [comment]
`ctrl-r<register>` with the prompt open

**Reply by KalinDimitrow** [reply]
Can I do the same in file picker or other pickers

**Reply by kirawi** [reply, marked as answer]
Yes, the popup with registers is just not visible.

**Reply by KalinDimitrow** [reply]
Thanks, its work. Is there any plans to make it more obvious

**Reply by sicher** [reply]
Vim has this really cool feature where you can type `ctrl-f` in command mode and it opens a buffer with the command history, that you can edit using the regular modes. So `p` would paste whatever is in the primary clipboard, for instance. It would be great to have something like that in helix as well, but I have no idea if it's feasible.
